### PR TITLE
update to latest version of bedrock autoloader

### PR DESCRIPTION
**Pull request by menno-ll**
## Issue link
 
https://app.asana.com/0/0/1206702425868151/f

## Description

Bedrock has added new features since v1.0.1, which we used as a base for our autoloader.
This PR adds the latest changes from bedrock, while keeping our own functions intact.

## How Has This Been Tested?

Placed it in Clarkson Theme Dawn

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: 0-loader.php [hunk 1]
```diff
@@ -70,46 +20,100 @@ function determine_autoload_dir(){
...
 
+        $this->llAutoload(); // Call the custom autoloading for Level Level
         $this->loadPlugins();
     }
 
+    /**
+     * Custom function by Level Level.
+     * This function returns the directory where the composer.json file is located.
+     */
+    public function llDetermineAutoloadeDir(): string {
```
Comment (Mike-Hermans) [inline 0-loader.php]:
```suggestion
    public function llDetermineAutoloadDir(): string {
```

File: 0-loader.php [hunk 2]
```diff
@@ -70,46 +20,100 @@ function determine_autoload_dir(){
...
+    }
+
+    /**
+     * Custom function by Level Level.
+     * This function loads the pre-autoload file.
+     * Then it loads the composer autoload file.
+     * Finally it loads the post-autoload file.
+     */
+    private function llAutoload(): void {
+        $autoload_dir = $this->llDetermineAutoloadeDir();
```
Comment (Mike-Hermans) [inline 0-loader.php]:
```suggestion
        $autoload_dir = $this->llDetermineAutoloadDir();
```

File: tests/AutoloadFileTest.php [hunk 3]
```diff
@@ -65,6 +64,6 @@ public function testCanDetermineAutoloadDir(array $definitions, string $result)
         foreach($definitions as $key => $value){
             define($key, $value);
         }
-        $this->assertEquals(determine_autoload_dir(), $result);
+        $this->assertEquals((new Autoloader())->llDetermineAutoloadeDir(), $result);
```
Comment (Mike-Hermans) [inline tests/AutoloadFileTest.php]:
```suggestion
        $this->assertEquals((new Autoloader())->llDetermineAutoloadDir(), $result);
```

**Review by Mike-Hermans — APPROVED** [review]
Handig dat je de ll-specifieke functies een namespace hebt gegeven.

**Review by NielsdeBlaauw — APPROVED** [review]

**Pull request merged**